elements/pl-matching/pl-matching.mustache

```diff
@@ -0,0 +1,110 @@
+<script>
+    $(function(){
+        $('#pl-matching-{{uuid}} [data-toggle="popover"]').popover({
+            sanitize: false,
+            container: 'body',
+            template: '<div class="popover pl-dropdown-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+    });
+</script>
+<style>
+.matching-options ol > li:before {
+    content: counter(item, {{counter_type}}) ".";
+}
+</style>
```

Comment:
This could be a minor issue if there is more than one `pl-matching` element in the same question, particularly if the counter type is different. Suggestion:
```suggestion
#pl-matching-{{uuid}} .matching-options ol > li:before {
    content: counter(item, {{counter_type}}) ".";
}
</style>
```